enhancements/sig-architecture/33-detached-deploy-mode/detached-deploy-mode.md

```diff
@@ -0,0 +1,217 @@
+| | klusterlet   |  | registration-agent |  | work-agent    |      |                                              
+| +------|-------+  +--------------------+  +-------|-------+      |                                              
+|        |                    |                     |              |                                              
++--------|--------------------|---------------------|--------------+                                              
+         |                    |                     |                                                             
+         |                    |                     |                                                             
+         ---------------------|----------------------                                                             
+                              | watch,create,update,delete...                                                     
+                              |                                                                                   
++-----------------------------v------------------------------------+                                              
+|  managed-cluster                                                 |                                              
+|                                                                  |                                              
+|  +----------------------+   +----------------------+             |                                              
+|  | CRDs:                |   | Configurations:      |             |                                              
+|  | clusterClaim         |   | role,rolebinding,sa  |   etc.      |                                              
+|  | appliedmanifestwork  |   |                      |             |                                              
+|  +----------------------+   +----------------------+             |                                              
++------------------------------------------------------------------+  
+```
+
+- The managed-cluster is the cluster managed by the hub cluster, manifestwork will be deployed onto this cluster.
+- The management-cluster is the cluster where Klusterlet related deployments are actually running.
+
+The managed cluster will not connect to the hub cluster, all things done by the management cluster. The management cluster could be the hub cluster, it could be another third-party cluster as well.
+
+In Detached mode, the "klusterlet.spec.namespace" will be ignored, all agent components will be installed to the namespace named <klusterlet's name>-open-cluster-management-agent. The other thing to note is that users are responsible for creating an external-managed-kubeconfig in the agent installation namespace, Klusterlet-operator will use this kubeconfig to get a new minimum permission kubeconfig for the managed cluster, then render it to registration-agent and work-agent.
```

Comment:
```suggestion
In Detached mode, the "klusterlet.spec.namespace" will be ignored, all agent components will be installed to the namespace with the same name as klusterlet. The other thing to note is that users are responsible for creating an `external-managed-kubeconfig` in the agent installation namespace, Klusterlet-operator will use this kubeconfig to get a new minimum permission kubeconfig for the managed cluster, then render it to registration-agent and work-agent.
```

Reply:
The commit is squshed to and sign information.